src/goInstallTools.ts

```diff
@@ -290,6 +290,11 @@ export function updateGoPathGoRootFromConfig(): Promise<void> {
 
 	// If GOPATH is still not set, then use the one from `go env`
 	let goRuntimePath = getGoRuntimePath();
+
```

Comment:
This could go into vscode-go. Have you considered posting a PR there? I try to keep the possibility to rebase without major problems for as long as possible.

Reply:
It is complicated for us (long/complicated/dumb corporate procedures) to contribute to other projects than those we've been specifically allowed to.  Can you open a PR yourself instead?